org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/completion/LSCompletionProposal.java

```diff
@@ -122,7 +123,7 @@ public class LSCompletionProposal
 	protected int bestOffset = -1;
 	protected int currentOffset = -1;
 	protected @Nullable ITextViewer viewer;
-	private final IDocument document;
+	protected final IDocument document;
```

Comment:
I am not following this part of the PR. Why does it need to be protected?

Reply:
Indeed... was a rogue change... :-\